tests/auth.test.js

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, test } from "@jest/globals";
+import request from "supertest";
+import app from "../src/app.js";
+
+const USERNAME = "user";
+const PASSWORD = "password";
+
+describe("does the jwt token exchange work", () => {
+	test("username and password to the right endpoint returns a token", async () => {
+    const response = await request(app)
+		.post("/api/token")
+    .set("Authentication", "Basic " + btoa(USERNAME + ":" + PASSWORD))
+    .send()
+		.expect(200);
+
+    expect(response.body).toHaveProperty("token");
+  })
+
+  test("wrong username and password returns 401", async () => {
+    const response = await request(app)
+		.post("/api/token")
+    .set("Authentication", "Basic " + btoa(USERNAME + ":" + PASSWORD + "wrong"))
+    .send()
+		.expect(401);
+    
+    expect(response.body.message).toContain("invalid")
+  })
+
+  test("review endpoint does not work without a proper auth header", async () => {
```

Comment:
This is a great test! The only problem is that it relies on that special `TEST` route, which decreases the reliability of the test (because it takes a special code path). Consider using a mocked data source in cases like this instead.